index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <main>
+        <div id="navdiv">
+        <ul>
+        <li ><a href="#Olasupo Ismail">Olasupo Ismail</a></li>
+        <li id="shift"><a href="#"><i class="fa-solid fa-envelope"></i></a></li>
+        <li id="shift"><a href="#Portfolio">Portfolio</a></li>
+        <li id="shift"><a href="#About">About</a></li>
+        <li id="shift"><a href="#Contact">Contact</a></li>
+        </ul>
+        </div>
+    </main>
+    <section>
+        <div class="one" style="font-size:1vw">
+        <header>Olasupo Ismail</header>
+        <h3>Software Developer</h3>
+        <br><br>
+        <p>I can help you build a product. feature or website, look through some of my work and experience! if you like what 
+        you see and have a project you need coded, don't hesitate to contact me</p> 
+        </div>
+        </section>
+        <section class="two">
+            <div style="overflow-x:auto;">
+            <table>
+            <tr>
+            <th>Languages</th>
+            <th>Framework</th>
+            <th>Skills</th>
+            </tr>
+            <tr>
+            <td>Javascript</td>
+            <td>Boostrap</td>
+            <td>Codekit</td>
+            </tr>
+            <tr>
+            <td>Ruby</td>
+            <td>Ruby on Rails</td>
+            <td>Github</td>
+            </tr>
+            <tr>
+            <td>HTML</td>
+            <td>RSPec</td>
+            <td>Codepen</td>
```

Comment:
it's nice using the table HTML semantic but I would suggest you have this section writing inform of a list tag